src/components/AuthenticatedAppHeader/NotificationsPane.jsx

```diff
@@ -117,7 +116,7 @@ export default function NotificationsPane({
                     }}
                   >
                     <div style={{ display: 'flex' }}>
-                      <Avatar src={shane} variant="circular" />
+                      <Avatar>{user1Name[0].toUpperCase()}</Avatar>
```

Comment:
`user1Name` should either be a non-empty value from the notification or "Unnamed User", so there should always be a first letter.
https://github.com/WildMeOrg/codex-frontend/blob/1fb6b1af5e84f92a6953666ff606826ed276e8c6/src/utils/notificationUtils.js#L10-L11